db/migrate/20160104131622_remove_deprecated_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveDeprecatedColumns < ActiveRecord::Migration
+  def change
+    remove_column :subject_workflow_counts, :set_member_subject_id
```

Comment:
these migrations will run before the prod code has updated to not use the columns right?